server/controllers/componentsController.test.ts

```diff
@@ -110,224 +44,95 @@ const defaultFooterViewModel = {
   ],
   isPrisonUser: true,
   component: 'footer',
-  services: defaultUserData.services,
 }
 
-const defaultMeta = {
-  activeCaseLoad: {
-    caseLoadId: 'LEI',
-    caseloadFunction: '',
-    currentlyActive: true,
-    description: 'Leeds',
-    type: '',
-  },
-  caseLoads: [
-    {
-      caseLoadId: 'LEI',
-      caseloadFunction: '',
-      currentlyActive: true,
-      description: 'Leeds',
-      type: '',
-    },
-  ],
-  services: [
-    {
-      description: '',
-      heading: 'Service',
-      href: '/href',
-      id: 'service',
-    },
-  ],
+const expectedMeta = {
+  activeCaseLoad: activeCaseLoadMock,
+  caseLoads: [activeCaseLoadMock],
+  services: servicesMock,
 }
 
-describe('getHeaderViewModel', () => {
-  it('should return the HeaderViewModel', async () => {
-    const output = await controller.getHeaderViewModel(defaultUser)
-    expect(output).toEqual(defaultHeaderViewModel)
-    expect(userServiceMock.getUserData).toBeCalledTimes(1)
+describe('componentsController', () => {
+  afterEach(() => {
+    jest.clearAllMocks()
   })
 
-  it('should return empty caseload information if not a nomis user', async () => {
-    const output = await controller.getHeaderViewModel({
-      ...defaultTokenData,
-      authSource: 'auth',
-      token: 'token',
-      roles: [],
-    })
-    expect(output).toEqual({
-      caseLoads: [],
-      changeCaseLoadLink: 'http://localhost:3001/change-caseload',
-      component: 'header',
-      ingressUrl: 'localhost',
-      isPrisonUser: false,
-      manageDetailsLink: 'http://localhost:9090/auth/account-details',
-      dpsSearchLink: 'http://localhost:3001/prisoner-search',
-      menuLink: 'http://localhost:3001#homepage-services',
-      activeCaseLoad: null,
-      services: [],
+  describe('getHeaderViewModel', () => {
+    it('should return the HeaderViewModel for a prison user', async () => {
+      const output = await controller.getHeaderViewModel(prisonUserMock)
+      expect(output).toEqual({ ...expectedHeaderViewModel, isPrisonUser: true })
     })
-  })
 
-  describe('user data is passed in', () => {
```

Comment:
No longer need to do this passing around of case load and service data since it's part of the user object now